src/HtmlConverter.php

```diff
@@ -190,6 +190,23 @@ protected function convertToMarkdown(ElementInterface $element)
     }
 
     /**
+     * Recursively decode html entities
+     *
+     * @param string $markdown
+     *
+     * @return string
+     */
+    public function recursivelyDecode($markdown)
+    {
+        $new_markdown = html_entity_decode($markdown, ENT_QUOTES, 'UTF-8');
+        if ($new_markdown === $markdown) {
+            return $new_markdown;
+        } else {
+            return $this->recursivelyDecode($new_markdown);
+        }
+    }
```

Comment:
While I do appreciate that you're trying to provide a built-in method for handling these cases, I'm not sure we should include it in this library (especially since the library won't even be using it).  I'd be okay adding a section to  `README.md` which includes this method (as a standalone function; i.e. without `$this->`)